ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.12-candidate-x86-loaded-upgrade-from-4.12.yaml

```diff
@@ -36,6 +36,7 @@ tests:
     cluster_profile: aws-perfscale-qe
     env:
       CHANNEL_GROUP: candidate
+      CHURN: "false"
```

Comment:
Disabling churning was missing in this job

Reply:
Would prefer to have unrelated changes in a new PR.